default/stringtables/en.txt

```diff
@@ -1227,6 +1227,18 @@ Show detailed empires data
 RULE_SHOW_DETAILED_EMPIRES_DATA_DESC
 Show research and production, researched buildings, parts, and hulls of all empires.
 
+RULE_DIPLOMACY
+Allowed diplomacy
+
+RULE_DIPLOMACY_DESC
+Manages diplomatic restrictions between players. By default all interactions are allowed.
+
+RULE_DIPLOMACY_ALLOWED_FOR_ALL
+Diplomacy allowed for all
+
+RULE_DIPLOMACY_FORBIDDEN_FOR_ALL
+Diplomacy forbidden for all
```

Comment:
`Always War` or similar

Reply:
I suppose it better to name `All forbidden` because for the same team #2542 it supposed to prevent breaking an alliance.

Reply:
I suppose it better to name `All forbidden` because for the same team #2542 it supposed to prevent breaking an alliance.

Reply:
It's not really clear from the name that it means "no changes after start" and not "always war between everyone"